.github/workflows/publish-rock.yaml

```
@@ -49,22 +47,25 @@ jobs:
           password: ${{ secrets.DOCKER_PASSWORD }}
 
       - name: Export ROCK to Docker
-        id: rock_in_docker
         run: |
-          COMMIT_ID=$(git rev-parse --short HEAD)
-          DOCKER_IMAGE=${{ inputs.rock-reference }}-$COMMIT_ID
-          sudo rockcraft.skopeo --insecure-policy copy oci-archive:${{ inputs.rock-filename }} docker-daemon:${{ inputs.registry }}/$DOCKER_IMAGE
-          echo "image=$DOCKER_IMAGE" >> "$GITHUB_OUTPUT"
+          echo "Exporting image to Docker: ${{ inputs.full-image-tag }}"
+          sudo rockcraft.skopeo --insecure-policy copy \
```

suggestion(non-blocking): We can run this without `sudo`, no?
```suggestion
          rockcraft.skopeo --insecure-policy copy \
```

---

Yeah without it i am being prompted for password while copying to docker registry locally

---

To not require the `sudo`, you have to include your $USER in the `docker` group as done [here](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user). Leaving this up to you for what makes more sense, leaving this unresolved just for visibility